### DISCUSSION TITLE
How to touch Mixcloud? The app does not seem to respond to mouse clicks

### DISCUSSION THREAD

**Discussion by mailinglists35** [PlayCover/PlayCover#1642] [Q&A]
macbook Air M1, dualbooting Sequoia and Ventura
I also tried holding Option and moving the floating circle down to the buttons then pressing the touchpad, does not respond neither
![image](https://github.com/user-attachments/assets/53147ae7-365f-4182-b32e-2d53390c942c)

**Comment by samxspam** [comment]
does Play Cover work on sequoia?

**Reply by mailinglists35** [reply]
yes. I am able to launch many other apps. Even eventually mixcloud received some mouse touches, but it is inconsistent, sometimes it works, sometimes not.
